app/[locale]/institute/administration/(committees)/board-of-governors/page.tsx

```diff
@@ -2,8 +2,12 @@ import Committee from '../committee';
 
 export default async function BoardOfGovernors({
   params: { locale },
+  searchParams,
 }: {
   params: { locale: string };
+  searchParams: Record<string, string | string[] | undefined>;
```

Comment:
??
```suggestion
  searchParams: { page: string; rowsPerPage: string };
```

Also, we won't let the user choose the number of rows per page.